CircuitPython_Essentials/CircuitPython_I2C_Scan/code.py

```diff
@@ -1,9 +1,4 @@
 """CircuitPython I2C Device Address Scan"""
-# If you run this and it seems to hang, try manually unlocking
-# your I2C bus from the REPL with
-#  >>> import board
-#  >>> board.I2C().unlock()
-
```

Comment:
Apologies for not being clear. Please leave it in this copy of the code, it is not explained in the Essentials guide, only in the template page.